frontend/src/main/components/Commons/CommonsTable.js

```diff
@@ -22,84 +24,83 @@ export default function CommonsTable({ commons, currentUser }) {
     // Stryker enable all
 
     // Stryker disable next-line all : TODO try to make a good test for this
-    const deleteCallback = async (cell) => { 
+    const deleteCallback = async (cell) => {
         console.log("deleteCallback cell=", cell);
-        deleteMutation.mutate(cell); 
-    }
+        deleteMutation.mutate(cell);
+    };
 
     const leaderboardCallback = (cell) => {
-        navigate(`/leaderboard/${cell.row.values["commons.id"]}`)
-    }
+        navigate(`/leaderboard/${cell.row.values["commons.id"]}`);
+    };
 
     const columns = [
         {
-            Header: 'id',
-            accessor: 'commons.id', // accessor is the "key" in the data
-
+            Header: "id",
+            accessor: "commons.id", // accessor is the "key" in the data
+        },
+        {
+            Header: "Name",
+            accessor: "commons.name",
         },
         {
-            Header:'Name',
-            accessor: 'commons.name',
+            Header: "Cow Price",
+            accessor: (row) => row.commons.cowPrice,
+            id: "commons.cowPrice",
         },
         {
-            Header:'Cow Price',
-            accessor: row => row.commons.cowPrice,
-            id: 'commons.cowPrice'
+            Header: "Price Change",
+            accessor: "commons.priceChange",
```

Comment:
Why does Price Change not need an id? Is that on purpose? Also, why is the accessor different from the others?

Reply:
I tried playing around with this, I believe this is correct as can be seen on lines 39 and 43 in the same file.